integration/revoke-document.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Selector } from "testcafe";
+import { enterPassword, loadConfigFile } from "./helper";
+import { homedir } from "os";
+import { join } from "path";
+import { existsSync, readFileSync } from "fs";
+import { file } from "jszip";
+
+fixture("Revoke flow").page`http://localhost:3000`;
+
+const Config = "./../src/test/fixtures/sample-config-local.json";
+const Title = Selector("h1");
+const Button = Selector("button");
+const ProgressBar = Selector("[data-testid='progress-bar']");
+const SubmitButton = Selector("[data-testid='form-submit-button']");
+const DownloadLink = Selector("[data-testid='download-file-button']");
+
+function getFileDownloadPath(fileName: string): string {
+  return join(homedir(), "Downloads", fileName);
+}
+
+// From https://stackoverflow.com/a/57624660/950462
+const waitForFileDownload = async (t: TestController, filePath: string): Promise<boolean> => {
+  // Timeout after 10 seconds
+  for (let i = 0; i < 100; i++) {
+    if (existsSync(filePath)) return true;
+    await t.wait(100);
+  }
+  return existsSync(filePath);
+};
```

Comment:
not used?

Reply:
i will double check again, if not used will remove it.